### PR TITLE
First Draft JWT examples

### LINKED ISSUE #719: JWT can be used with cookie and query param is not documented
Also, check if other auth options are relevant as well

### PR DESCRIPTION
**Pull request by marksou**
Fixes #719

### CODE REVIEW

File: tyk-docs/content/security/your-apis/json-web-tokens.md [hunk 1]
```diff
@@ -145,9 +145,67 @@ You can now configure JWT clock skew using the following variables. All values a
...
 
+## JWT Examples
+
+Below are some example uses of JWT with Tyk.
+
+### JWT with Header
+
+1. Create an API from the Designer. For this example `jwt-test`
+2. In the **Authentication** settings, select JSON Web Token (JWT)as in [Step 1](/docs/security/your-apis/json-web-tokens/#step-1-set-authentication-mode) above
+3. For the JWT Signing method, select HAMAC from the drop-down list as in [Step 2](/docs/security/your-apis/json-web-tokens/#step-2-set-the-jwt-signing-method) above.
```
Comment (letzya) [inline tyk-docs/content/security/your-apis/json-web-tokens.md]: not hamac

Comment (letzya) [reply tyk-docs/content/security/your-apis/json-web-tokens.md]: We should say which hmac we are using

File: tyk-docs/content/security/your-apis/json-web-tokens.md [hunk 2]
```diff
@@ -145,9 +145,67 @@ You can now configure JWT clock skew using the following variables. All values a
...
+4. Enter `sub` as the Identity Source, as in [Step 3](/docs/security/your-apis/json-web-tokens/#step-3-set-the-identity-source-and-policy-field-name) above. The Policy field can be left blank.
```
Comment (letzya) [inline tyk-docs/content/security/your-apis/json-web-tokens.md]: Change to
“Set policy field to the name of the claim you will be using in your jwt when referring to the policy. For instance “pol” “

File: tyk-docs/content/security/your-apis/json-web-tokens.md [hunk 3]
```diff
@@ -145,9 +145,67 @@ You can now configure JWT clock skew using the following variables. All values a
...
+5. In the **Endpoint Designer**, enter a GET endpoint. For this example `ip`.
+6. Click Create to save the API.
+7. Edit the API and copy the API URL from the top of the screen.
+8. Enter the following cURL command. ```curl -k https://tyk123.cloudv2.tyk.io/jwt-test/ip```
```
Comment (letzya) [inline tyk-docs/content/security/your-apis/json-web-tokens.md:167]: Remove tyk123 and put a placeholder “{tyk-gateway}

File: tyk-docs/content/security/your-apis/json-web-tokens.md (hunk 1 above)
Comment (letzya) [reply tyk-docs/content/security/your-apis/json-web-tokens.md]: “Create an API from the Designer. For this example `jwt-test`” —> I don’t understand this wording. Did you mean to write “Create an API using the UI and name it `jwt-test`”
@mark

File: tyk-docs/content/security/your-apis/json-web-tokens.md [hunk 4]
```diff
@@ -145,9 +145,67 @@ You can now configure JWT clock skew using the following variables. All values a
 "jwt_not_before_validation_skew": 0
 ```
 
+## JWT Examples
+
+Below are some example uses of JWT with Tyk.
+
+### JWT with Header
```
Comment (letzya) [inline tyk-docs/content/security/your-apis/json-web-tokens.md]: Jwt with header?
Is it “Authenticate an api with user’s unique signing jwt”?

**Comment by letzya** [comment]
@marksou

**Review by letzya — CHANGES_REQUESTED** [review]

File: tyk-docs/content/security/your-apis/json-web-tokens.md (hunk 1 above)
Comment (marksou) [reply tyk-docs/content/security/your-apis/json-web-tokens.md]: Have changed this

File: tyk-docs/content/security/your-apis/json-web-tokens.md (hunk 2 above)
Comment (marksou) [reply tyk-docs/content/security/your-apis/json-web-tokens.md]: Added

File: tyk-docs/content/security/your-apis/json-web-tokens.md (hunk 3 above)
Comment (marksou) [reply tyk-docs/content/security/your-apis/json-web-tokens.md:167]: As we have used **tyk123** to create the example, this should stay in the command to allow the example to work.

File: tyk-docs/content/security/your-apis/json-web-tokens.md (hunk 4 above)
Comment (marksou) [reply tyk-docs/content/security/your-apis/json-web-tokens.md]: Changed

**Pull request closed without merging**